app/controllers/AccountManagement.scala

```diff
@@ -396,4 +412,11 @@ object AccountManagement extends Controller with ContextLogging with CatalogProv
     implicit val resolution: TouchpointBackend.Resolution = TouchpointBackend.forRequest(PreSigninTestCookie, request.cookies)
     ManageWeekly.renewThankYou
   }
+
+  def trackDelivery: Action[TrackDelivery] = accountManagementAction.async(parse.form(TrackDeliveryForm.lookup)) { implicit request =>
+    implicit val resolution: TouchpointBackend.Resolution = TouchpointBackend.forRequest(PreSigninTestCookie, request.cookies)
+    val lookup = request.body
+    ManageDelivery.fulfilmentCheck(lookup)
```

Comment:
inline this - as the top level def is ideally responsible for turning a web request into some nice scala object, and some nice scala object back into a web response.  as it is we're splitting that in half so the function signature of fulfilmentCheck is kind of crossing the abstraction levels